src/server/controllers/users.controller.js

```diff
@@ -0,0 +1,38 @@
+'use struct'
+var User = require('mongoose').model('User')
```

Comment:
Hmm I'm not sure but it seems many people are using upper case at first letter for collection name.